.github/workflows/cypress.yaml

```
@@ -8,17 +8,45 @@
   workflow_dispatch:
 
 jobs:
-  cypress-run:
-    permissions: write-all
+  install:
     if: github.repository == 'ohcnetwork/care_fe'
-    runs-on: ubuntu-24.04-arm
+    permissions:
+      write-all: true
```

actionlint (1.7.4)

unknown permission scope "write-all". all available permission scopes are "actions", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)